src/js/index.js

```diff
@@ -513,16 +527,31 @@ export const hpe = deepFreeze({
     },
   },
   tab: {
-    border: {
-      color: 'text-xweak',
+    color: 'text',
+    active: {
+      background: 'background-contrast',
     },
-    color: 'text-xweak',
-    margin: {
-      horizontal: 'none',
+    hover: {
+      background: 'background-contrast',
+      color: 'text',
     },
-    pad: {
-      horizontal: 'small',
+    border: {
+      side: 'bottom',
+      color: 'border',
+      active: {
+        color: 'border-strong',
+      },
+      hover: {
+        color: 'border',
+      },
     },
+    pad: 'small',
+    margin: 'none',
+    extend: ({ theme }) => css`
```

Comment:
Would be nice to not need `extend` for HPE theme. Not part of this PR per-se, but something we should look into. Can we add an issue to evaluate the use of `extend` in the HPE NEXT theme, to see if anything could/should be easier.